readme.md

```diff
@@ -66,8 +66,10 @@ They also try to be very welcoming for 1st-time contributors.
 ([all JavaScript projects with hacktoberfest issues](https://github.com/search?utf8=%E2%9C%93&q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+language%3AJava&type=Issues&ref=advsearch&l=JavaScript))
 
 - [jest-extended](https://github.com/mattphillips/jest-extended)
+- [letra-extension](https://github.com/jayehernandez/letra-extension)
```

Comment:
Where is the description gone? In your first proposal, you had a description, which was good as far as I remember.